build_deploy.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -ex
+
+QUAY="quay.io/app-sre"
+VERSION=`grep 'ENV SENTRY_VERSION' Dockerfile | cut -d' ' -f 3 | tr -d '\r\n'`
```

Comment:
we use the git commit hash as the image tag. take a look at https://github.com/app-sre/visual-qontract/blob/master/build_deploy.sh

Reply:
this is very important, thanks for catching this @maorfr